packages/dcos-integration-test/extra/test_dcos_log.py

```diff
@@ -356,6 +359,9 @@ def test_log_v2_pod_logs(dcos_api_session):
         check_response_ok(response, {})
         assert 'STDERR_LOG' in response.text, "Expect STDERR_LOG in stdout file. Got {}".format(response.text)
 
+        _browse_files(dcos_api_session, pod_id, ['stdout', 'stderr', 'foo'])
+        _download_files(dcos_api_session, pod_id, ['stdout', 'stderr', 'foo'])
```

Comment:
It looks like these functions are asserting some fact about the logs API. Can we name these functions according to that assertion, or explain what they're asserting in their docstrings? I can't tell what `_browse_files()` is guaranteeing from its name, and it doesn't have a docstring, so I have to read its code to know what it's verifying, which isn't ideal. A better name for `_browse_files()` might be e.g. `_assert_files_in_browse_response()`.

Reply:
fixed